source/documentation/index.md

```diff
@@ -27,9 +27,19 @@ When you are ready to test your software, first read and understand the [CTC Tra
 
 We strongly advise you to start testing your software for compatibility purposes as soon as possible.
 
+## What is Trader Test?
+
+Trader Test simulates automated responses. It also provides an environment in which NCTS support staff simulate the tasks done by Border Force personnel.
+
```

Comment:
One minor issue that arises only because you added this new section (What is Trader Test?). Below in the next section (What is the CTC Traders Test Support API?), you spell out the NCTS abbreviation. However, NCTS is first used in this section. If you want to keep `New Computerised Transit System (NCTS)`, it would be better to move it here and just use `NCTS` in the other section.

Reply:
Done.